src/tv2-common/blueprintConfig.ts

```diff
@@ -60,6 +60,7 @@ export interface TableConfigGfxSchema {
 }
 
 export interface TableConfigGfxSetup {
+	_id: string
```

Comment:
Since all(?) table config items have this `_id`, we could probably make more of those interfaces all extend a base one, but I guess that can also be done when we start using DefaultSchema and DefaultDesign